jetty-maven-plugin/src/main/java/org/eclipse/jetty/maven/plugin/JettyDeployWar.java

```diff
@@ -56,19 +55,13 @@
      * @parameter  default-value="true"
      */
     protected boolean daemon = true;
-    
-    
+
```

Comment:
There's only white space changes in  this file, can you take it out of the commit list, and the same for ServerConnectorListener?

Reply:
I will squash everything in a single commit. Does it work for you? 
Otherwise I can remove those cosmetics changes :-)

Reply:
I'd prefer the whitespace changes weren't part of the commit at all.